src/StakedCitadel.sol

```diff
@@ -903,22 +900,22 @@ contract StakedCitadel is
             uint256 feeGovernance,
             uint256 feeStrategist
         ) = _calculatePerformanceFee(_harvestedAmount);
-        uint256 duration = harvestTime.sub(lastHarvestedAt);
+        uint256 duration = harvestTime - lastHarvestedAt;
 
         // Management fee is calculated against the assets before harvest, to make it fair to depositors
         uint256 management_fee = managementFee > 0
-            ? managementFee
```

Comment:
This code is shown as going from:
`managementFee.mul(balance().sub(_harvestedAmount)).mul(duration).div(SECS_PER_YEAR).div(MAX_BPS)`
to 
`managementFee * balance() - (_harvestedAmount * duration) / SECS_PER_YEAR / MAX_BPS`

however, is:
`managementFee * (balance() - _harvestedAmount) * duration / SECS_PER_YEAR / MAX_BPS` not the correct way to translate?